m4/ax_python_module.m4

```diff
@@ -21,13 +23,13 @@
 #   and this notice are preserved. This file is offered as-is, without any
 #   warranty.
 
-#serial 6
+#serial 7
 
 AU_ALIAS([AC_PYTHON_MODULE], [AX_PYTHON_MODULE])
 AC_DEFUN([AX_PYTHON_MODULE],[
     if test -z $PYTHON;
     then
-        PYTHON="python"
+        PYTHON="$3"
```

Comment:
Well, the synopsis suggests that the parameter is optional, so IMHO the macros should do something useful even if the 3rd parameter is left unspecified.